Makefile

```diff
@@ -35,8 +35,8 @@ GOBIN=$(shell go env GOBIN)
 endif
 
 CONTAINER_ENGINE ?= docker
-IMAGE_HOST ?= quay.io
-IMAGE_NAMESPACE ?= shipwright
+IMAGE_HOST ?= ghcr.io
+IMAGE_NAMESPACE ?= shipwright-io/operator
```

Comment:
feels more accurate to leave IMAGE_NAMESPACE as shipwright-io, but then create a new IMAGE_NAME var whose default is operator

then set IMAGE_REPO ?= $(IMAGE_HOST)/$(IMAGE_NAMESPACE)/$(IMAGE_NAME)

Reply:
The full image ref when `make ko-publish` is invoked becomes `ghcr.io/shiprwright-io/operator/operator:${VERSION}`. Unlike Quay or Docker Hub, GitHub packages have two levels of organization (organization/repo) instead of one (organization or user ID).

I'd rather keep one variable so that if a user wants to publish to their own fork of the operator, they just run `make ko-publish IMAGE_NAMESPACE=<my-github-username>/<my-github-repo>`

Reply:
ok sure the `ko` element conceivably alters the usability of the semantics ... what you articulated seems reasonable;
certainly easy enough to change as we collectively get more involved and if non `ko` workflows ever emerge.

Since @otaviof just lgtm'ed this @adambkaplan - I think you are set with either self approving or getting one of the other folks from the approver list to approve.